doc/python/configuration-options.md

```diff
@@ -33,40 +33,150 @@ jupyter:
+The `show()` method that you use to display your figures also accepts a `config` parameter.
+
+You can set the configuration options for your figure by passing a dictionary to this parameter which contains the options you want to set. 
+
+If you don't set an option's value, it will be automatically be set to the default value for that option. 
+
+For the complete list of configuration options and their defaults see: https://github.com/plotly/plotly.js/blob/master/src/plot_api/plot_config.js
 
 ##### Enable Scroll Zoom
 
+This option allows users to zoom in and out of figures using the scroll wheel on their mouse and/or a two-finger scroll. 
+
 ```python
 import plotly.graph_objects as go
 
 fig = go.Figure()
 
+config = dict({'scrollZoom': True})
+
 fig.add_trace(
     go.Scatter(
         x=[1, 2, 3],
         y=[1, 3, 1]))
 
-fig.show(config={'scrollZoom': True})
+fig.show(config=config)
 ```
 
-##### Display ModeBar
+##### Force The Modebar to Always Be Visible
+
+When users hover over a figure generated with plotly.py, a modebar appears in the top-right of the figure. This presents users with several options for interacting with the figure.
+
+By default, the modebar is only visible while the user is hovering over the chart. If you would like the modebar to always be visible regardless of whether or not the user is currently hovering over the figure, set the displayModeBar attribute in the configuration of your figure to true.
 
 ```python
 import plotly.graph_objects as go
 
 fig = go.Figure()
 
+config = {'displayModeBar': True}
+
 fig.add_trace(
     go.Scatter(
         x=[1, 2, 3],
         y=[1, 3, 1]))
 
-fig.show(config={'displayModeBar': True})
+fig.show(config=config)
 ```
 
-##### Edit Mode - change the title and axis titles
+##### Never Display The Modebar
+
+When users hover over a figure generated with `plotly.py`, a modebar appears in the top-right of the figure. This presents users with several options for interacting with the figure.
+
+By default, the modebar is only visible while the user is hovering over the chart. If you would like the modebar to never be visible, then set the displayModeBar attribute in the config of your figure to false.
+
+```python
+import plotly.graph_objects as go
+
+fig = go.Figure()
+
+config = {'displayModeBar': False}
+
+fig.add_trace(
+    go.Scatter(
+        x=[1, 2, 3],
+        y=[1, 3, 1]))
+
+fig.show(config=config)
+```
+
+##### Display the `Edit Chart` Link
+
+Set `showLink` to `True` in order to make your figure editable on [Chart Studio](https://plot.ly/online-chart-maker). 
+
+```python
+import plotly.graph_objects as go
+
+fig = go.Figure()
+
+config = {'showLink': True}
+
+fig.add_trace(
+    go.Scatter(
+        x=[1, 2, 3],
+        y=[1, 3, 1]))
+
+fig.show(config=config)
+```
+
+### Display The `Edit In Chart Studio` Modebar Button
```

Comment:
Not clear to me how this is different from the previous example?

Reply:
![Screen Shot 2020-02-17 at 11 40 56 AM](https://user-images.githubusercontent.com/1557650/74672099-8e17b600-517a-11ea-9cb9-0b26c20c1c9a.png)

 This example has a button in the modebar rather than the hyperlink in the bottom right of the figure.

Reply:
ok, maybe add a sentence here and above, and/or have the two examples in the same section?